src/main/java/jenkins/branch/BranchEventCause.java

```diff
@@ -75,6 +78,11 @@ public String getOrigin() {
         return origin;
     }
 
+    @NonNull
```

Comment:
Old causes will have a null event, this is going to have to be `@CheckForNull`

Reply:
```suggestion
    @CheckForNull
```